rpcd/etc/openstack_deploy/user_rpco_variables_overrides.yml.aio

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright 2016-2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Using this file
+#
+# This file is laid down only once and is used for overriding variables based on the file name.
+# This file will not be overwritten by upgrades.
+
+elasticsearch_heap_size_mb: 1024
+lb_name: "{{ ansible_hostname }}"
+maas_notification_plan: "npTechnicalContactsEmail"
+net_max_speed: 1000
+
+{% if (groups['ceph_all'] is defined) and (groups['ceph_all'] | length > 0) %}
```

Comment:
This won't work as the dynamic inventory is not being used when this file is templated out.